### ISSUE TITLE
In timeline, 'Add block' and 'Delete week' links are not accessible via screen reader

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#1018]
Because they attach to elements that screen readers don't think are clickable, they don't work for JAWS (at least).

They should be done with normal links or buttons, without changing the current visual style.

**Comment by ragesoss** [comment]
Confirmed fixed!
